### PR TITLE
Add jobs section with a job

### LINKED ISSUE #1411: English job(s)
I dont like that we have to link to Dutch website to link to a job. Atleast 1 dev job should be on the English website

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Add translated text for index 
- [x] Add translated job Elixir 
- [x] Fix Homerun Endpoint
- [x] Improve copy

Close #1411

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
@sn3p @lennartdefacto @SusanDefacto can you please check the English copy in this Pull Request? Thanks in advance 👍

File: locales/en.yml [hunk 1]
```diff
@@ -224,6 +224,13 @@ en:
         title: CAPP Quality Portfolio
   implementaties:
     heading: Implementation doesn’t have to be complicated
+  jobs:
+    acquisition: "Note: acquisition based on this vacancy is not appreciated."
+    apply: "Apply now!"
+    header: "Defacto as your employer"
```
Comment (sn3p) [inline locales/en.yml]: `Defacto as your employer` does not sound to inspiring (same goes for Dutch page).
Maybe think of something more engaging?

File: source/jobs/_jobs-text.en.md.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Nice to meet you! Maybe this is the beginning of a shared future.
+
+Engaged employees with innovative and creative ideas contribute to the success of an organization. Our credo is "Developing People", which is why we invest in our developers, consultants, sales and support staff.
```
Comment (sn3p) [inline source/jobs/_jobs-text.en.md.erb]:
```
... developers, consultants, sales and support staff.
```

Want to explicitly list all the teams here?

File: locales/en.yml (hunk 1 above)
Comment (RonaldDijkstra) [reply locales/en.yml]: Get in there Lennart 🥇 #1357

File: source/jobs/_jobs-text.en.md.erb (hunk 2 above)
Comment (RonaldDijkstra) [reply source/jobs/_jobs-text.en.md.erb]: ...which is why we invest in our team?

File: source/jobs/_jobs-text.en.md.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+Every employee as a link in the chain ensures that our clients receive great assistance and support and enables our family business to grow.
```
Comment (sn3p) [inline source/jobs/_jobs-text.en.md.erb]: Hard sentence to read..
Doesn't really cover what we're doing I think (sounds a bit like we are a callcenter)
Personally I would't use a growing `family business` here, sound like mafia 🤔Also growth is not a main goal I think?

Comment (RonaldDijkstra) [reply source/jobs/_jobs-text.en.md.erb]: 😅

Comment (RonaldDijkstra) [reply source/jobs/_jobs-text.en.md.erb]: But we do make offers they can't refuse. 🐴

Comment (RonaldDijkstra) [reply source/jobs/_jobs-text.en.md.erb]: We got a vacancy for this right? 

https://nl.wikipedia.org/wiki/Consigliere

File: locales/en.yml [hunk 4]
```diff
@@ -207,7 +207,7 @@ en:
     open-source: "Open Source"
   home:
     hero:
-      text: "More than 200.000 professionals use our software to learn and develop their talents. "
+      text: "More than 250.000 professionals use our software to learn and develop their talents. "
```
Comment (RonaldDijkstra) [inline locales/en.yml]: This can or should be in another PR ;-)

File: source/jobs/_jobs-text.en.md.erb (hunk 3 above)
Comment (RonaldDijkstra) [reply source/jobs/_jobs-text.en.md.erb]: JOIN THE DEFACTO FAMILY.

**Comment by lennartdefacto** [comment]
Only updated the main english job text, was thinking about implementing the feedback for the developer vacancy in seperate PR, as Ronald suggested a while ago.

**Comment by RonaldDijkstra** [comment]
Needs a rebase I see

**Comment by RonaldDijkstra** [comment]
No there is not 😆

**Comment by RonaldDijkstra** [comment]
This is ready
